content/en/graphing/faq/why-does-zooming-out-a-timeframe-also-smooth-out-my-graphs.md

```diff
@@ -1,20 +1,28 @@
 ---
 title: Why does zooming out a timeframe also smooth out my graphs?
 kind: faq
+disable_toc: true
+further_reading:
+- link: "/developers/metrics/types/"
+  tag: "Documentation"
+  text: "Discover Datadog metrics types"
+- link: "/graphing/functions/rollup/"
+  tag: "Documentation"
+  text: "Learn more about the rollup function"
 ---
-Within Datadog, a graph can only contain a set number of points and, as the timeframe over which a metric is viewed increases, aggregation between points occurs to stay below that set number. Thus you will lose in granularity as you increase the timeframe. For instance, for a 4-hour time window, data is aggregated to have one value per minute. As you "zoom out" (i.e. select a larger timeframe) the data shown on the graph will represent larger time period.
 
-You can append the [`.rollup()` function][1] to your query to adjust the method and granularity of time aggregation. Datadog rolls up data points automatically, based on the in-app metric type: `gauge` metrics are averaged, whereas `count` and `rate` metrics are summed. If you wanted to aggregate the sum of the metric over a one day period, you could append .rollup(sum, 86400) to your query. If you want to keep an eye on the max values, you may use the maximum aggregation .rollup(max).
+Within Datadog, a graph can only contain a set number of points and, as the timeframe over which a metric is viewed increases, aggregation between points occurs to stay below that set number. Thus you lose in granularity as you increase the timeframe. For instance, for a 4-hour time window, data is aggregated to have one value per minute for a line graph, and one value per two minutes for a bar graph. As you "zoom out" (i.e. select a larger timeframe) the data shown on the graph will represent larger time period.
```

Comment:
```suggestion
Within Datadog, a graph can only contain a set number of points and, as the timeframe over which a metric is viewed increases, aggregation between points occurs so that the number of points remains under that set number. Thus, granularity is lost as the timeframe increases. For instance, for a four hour time window, data is aggregated to have one value per minute for a line graph, and one value per two minutes for a bar graph. As you "zoom out" (i.e. select a larger timeframe) the data shown on the graph represents a longer time period.
```